android/src/main/java/com/reactlibrary/YMChatService.java

```diff
@@ -40,8 +40,12 @@ public void setBotId(String botId) {
         ymChat.config.payload = payloadData;
     }
 
-    public void startChatbot(Context context) throws Exception {
-        ymChat.startChatbot(context);
+    public void startChatbot(Context context) {
+        try {
+            ymChat.startChatbot(context);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
How did we miss this in the last release?

Reply:
The app was built and tested from React CLI itself. So we might have missed it.